### DISCUSSION TITLE
Change line/plot/background color?

### DISCUSSION THREAD

**Discussion by hauselin** [leonhma/obsidian-functionplot#25] [General]
Hi, thanks for the great plugin! Wonder if it's possible to change the default line/plot/background colors? Thanks!

**Comment by leonhma** [comment]
Not currently. As far as I have tested, the plot adjusts to light/dark mode quite well. You can open a feature request if you need configurable colors. I gotta see how I can integrate that into the library I am using to create the plots.

**Reply by hauselin** [reply]
If the colors are hard to tweak/adjust, would it be easier to increase the default thickness of the lines and the font size to increase readability? Right now, I find the axis labels and lines/plots a little small. Though not sure if it's my setup/theme?

**Reply by leonhma** [reply]
I definitely agree. The library I am using doesn't let me adjust this though. So that's something I'll have to figure out. Can you open a feature request for that?

**Comment by leonhma** [comment]
Just as a follow-up for anyone else looking. The rendered plot is now customizable via options such as color, font size and line width in the plugin's settings. This feature is available on `1.2.0` and later.

**Reply by hauselin** [reply]
Awesome! Thanks!
